pkg/noderesourcetopology/filter.go

```diff
@@ -41,74 +42,71 @@ const highestNUMAID = 8
 
 type PolicyHandler func(pod *v1.Pod, zoneMap topologyv1alpha2.ZoneList) *framework.Status
 
-func singleNUMAContainerLevelHandler(pod *v1.Pod, zones topologyv1alpha2.ZoneList, nodeInfo *framework.NodeInfo) *framework.Status {
-	klog.V(5).InfoS("Single NUMA node handler")
+func singleNUMAContainerLevelHandler(lh logr.Logger, pod *v1.Pod, zones topologyv1alpha2.ZoneList, nodeInfo *framework.NodeInfo) *framework.Status {
+	lh.V(5).Info("Single NUMA node handler")
 
 	// prepare NUMANodes list from zoneMap
-	nodes := createNUMANodeList(zones)
+	nodes := createNUMANodeList(lh, zones)
 	qos := v1qos.GetPodQOS(pod)
 
 	// Node() != nil already verified in Filter(), which is the only public entry point
-	logNumaNodes("container handler NUMA resources", nodeInfo.Node().Name, nodes)
+	logNumaNodes(lh, "container handler NUMA resources", nodeInfo.Node().Name, nodes)
 
 	// the init containers are running SERIALLY and BEFORE the normal containers.
 	// https://kubernetes.io/docs/concepts/workloads/pods/init-containers/#understanding-init-containers
 	// therefore, we don't need to accumulate their resources together
 	for _, initContainer := range pod.Spec.InitContainers {
-		logID := fmt.Sprintf("%s/%s/%s", pod.Namespace, pod.Name, initContainer.Name)
-		klog.V(6).InfoS("target resources", stringify.ResourceListToLoggable(logID, initContainer.Resources.Requests)...)
+		lh.V(6).Info("target resources", stringify.ResourceListToLoggable(initContainer.Resources.Requests)...)
```

Comment:
maybe reword `target resources` to `init container resources` ?

Reply:
thanks, rephrased